packages/salesforcedx-vscode-apex/src/index.ts

```diff
@@ -52,9 +52,11 @@ export async function activate(context: vscode.ExtensionContext) {
     .then(async () => {
       LanguageClientUtils.languageClientReady = true;
       await testOutlineProvider.refresh();
+      telemetryService.sendApexLSPActivationEvent(langClientHRStart);
```

Comment:
Time reported is different, but I think more accurate, than the once reported on Output tab in vscode by the lsp.

Reply:
I think this time will take into account the _whole_ initialization cycle. The one at the output only considers the startup time from jorje. Also, remember, jorje does some things in an async way (by design so that we don't block), so the time reported might still be different.

For the large code base sample I sent, what is the difference?

Reply:
Apex LSP returns 2 times (Scanning built-in and system Apex types& Scanning user-defined types). I'm comparing against scanning user-defined types. For small/medium size projects the difference in metrics is 1-2 seconds. For the large project I'm getting a significant discrepancy e.g. metric ~34s vs jorje metric ~80s.

Reply:
The larger measure is also useful to know since that is the total time for Apex jorje to complete. It's not the full time for the user since Apex jorje asyncs some of the work (and doesn't block the UI). But, that number is good to record if possible.

We could use the number you are able to obtain more easily for now (but we need to interpret it with those caveats).